pkg/apis/pingcap/v1alpha1/backup.go

```diff
@@ -201,6 +201,22 @@ func IsBackupScheduled(backup *Backup) bool {
 	return condition != nil && condition.Status == corev1.ConditionTrue
 }
 
+// HaveTruncateUntil returns true if a Backup has truncate until set
+func HaveTruncateUntil(backup *Backup) bool {
+	if backup.Spec.Mode != BackupModeLog {
+		return false
+	}
+	return backup.Spec.LogTruncateUntil != ""
+}
+
+func IsBackupPaused(backup *Backup) bool {
+	if backup.Spec.Mode == BackupModeLog {
+		return IsLogBackupSubCommandOntheCondition(backup, BackupPaused)
```

Comment:
Seems log backup sub command condition has no `BackupPaused` status

Reply:
But it do have, after adding the pause stage and command in this PR